src/test/java/tests/application/AqualityServicesTests.java

```diff
@@ -0,0 +1,51 @@
+package tests.application;
+
+import aquality.selenium.core.logging.Logger;
+import com.google.inject.ConfigurationException;
+import com.google.inject.Injector;
+import org.testng.annotations.Test;
+import static org.testng.Assert.*;
+
+public class AqualityServicesTests{
+
+    @Test
+    public void testShouldBePossibleToGetDefaultInjector() {
+        Injector injector = CustomAqualityServices.getInjector();
+        assertNotNull(injector, "Default injector should not be null");
+
+        Logger logger = injector.getInstance(Logger.class);
+        assertNotNull(logger, "Logger should not be null");
+        assertThrows(ConfigurationException.class, () -> injector.getInstance(ICustomDependency.class));
+
+        Injector newInjector = CustomAqualityServices.getInjector();
+        assertEquals(injector, newInjector, "AqualityServices should return instance of injector");
+    }
+
+    @Test
+    public void testShouldBePossibleToGetCustomModule() {
+        CustomAqualityServices.setInjector(new TestModule());
+        Injector injector = CustomAqualityServices.getInjector();
+        assertNotNull(injector, "Custom injector should not be null");
+
+        ICustomDependency customDependency = injector.getInstance(ICustomDependency.class);
+        assertNotNull(customDependency, "ICustomDependency should not be injected in custom module");
```

Comment:
maybe 'should BE injected' in the message?

Reply:
done